build/wide-github.user.js

```diff
@@ -43,6 +45,14 @@ var styleSheet = "" +
   "padding-right: 0px !important;" +
 "}" +
 
+// CollinChaffin Fix for Octotree this also introduces unwanted horizontal scrollbars due to inline GH styles
```

Comment:
Attribution is usually done through GH commits... should probably omit all personal references outside of the metadata blocks.